tests/driver_bmp180/main.c

```diff
@@ -73,11 +77,12 @@ int main(void)
         /* Get altitude in meters */
         int16_t altitude = bmp180_altitude(&dev, pressure_0);
 
-        printf("Temperature [°C]: %d.%d\n"
+        printf("Temperature [°C]: %c%d.%d\n"
                "Pressure [hPa]: %lu.%d\n"
                "Pressure at see level [hPa]: %lu.%d\n"
               "Altitude [m]: %i\n"
                "\n+-------------------------------------+\n",
+               (negative ? '-' : ' '),
                (int)(temperature / 10), (int)(temperature % 10),
```

Comment:
```suggestion
                (int)(temperature / 10), abs(temperature % 10),
```

No need to introduce an extra variable here (and manually add the sign)

Reply:
Fixed